tools/appveyor-scripts/build.ps1

```diff
@@ -19,15 +19,24 @@ function PushAndroidArtifacts
 If ($env:build_type -eq "android_cpp_tests") {
     Write-Host "Build tests\cpp-tests"
     Push-Location $env:APPVEYOR_BUILD_FOLDER\tests\cpp-tests\proj.android\
-    & ./gradlew assembleDebug
+    & ./gradlew assembleDebug -PPROP_BUILD_TYPE=ndk-build --parallel --info
```

Comment:
Could you do cmake for android cop test and ndk build for Android Lia?

Reply:
yep, a little things